spec/values/calculation/abs.hrx

```diff
@@ -0,0 +1,255 @@
+}
+
+<===> operation/integer/warning
+DEPRECATION WARNING: Passing percentage units to the global abs() function is deprecated.
+In the future, this will emit a CSS abs() function to be resolved by the browser.
+To preserve current behavior: math.abs(85%)
+To emit a CSS abs() now: abs(#{85%})
+More info: https://sass-lang.com/d/abs-percent
+  ,
+1 | a {b: abs(17 * 5%)}
+  | ^^^^^^^^^^^^^^^^^^^
+  '
+    input.scss 1:1  root stylesheet
+
+<===>
+================================================================================
+<===> preserved/operation/percentage/input.scss
+a {b: abs(17px + 5%)}
+
+<===> preserved/operation/percentage/output.css
+a {
+  b: abs(17px + 5%);
+}
+
+<===>
+================================================================================
+<===> preserved/variable/input.scss
+a {
+  --test: 5;
+  b: abs(var(--test))
+}
+
+<===> preserved/variable/output.css
+a {
+  --test: 5;
+  b: abs(var(--test));
+}
+
+<===>
+================================================================================
+<===> preserved/unquoted_string/input.scss
+a {
+  b: abs(test);
+}
+
+<===> preserved/unquoted_string/output.css
+a {
+  b: abs(test);
+}
+
+<===>
+================================================================================
+<===> named_argument/input.scss
+a {b: abs($number: -3)}
+
+<===> named_argument/output.css
+a {
+  b: 3;
+}
+
+<===>
+================================================================================
+<===> abs_percentage_warning/input.scss
+a {b: abs(-7.5%)}
+
+<===> abs_percentage_warning/output.css
+a {
+  b: 7.5%;
+}
+
+<===> abs_percentage_warning/warning
+DEPRECATION WARNING: Passing percentage units to the global abs() function is deprecated.
+In the future, this will emit a CSS abs() function to be resolved by the browser.
+To preserve current behavior: math.abs(-7.5%)
+To emit a CSS abs() now: abs(#{-7.5%})
+More info: https://sass-lang.com/d/abs-percent
+  ,
+1 | a {b: abs(-7.5%)}
+  | ^^^^^^^^^^^^^^^^^
+  '
+    input.scss 1:1  root stylesheet
+
+<===>
+================================================================================
+<===> math/slash_as_division/input.scss
+b {
+  a: 2px / abs(1.5);
+}
+
+<===> math/slash_as_division/output.css
+b {
+  a: 1.3333333333px;
+}
+
+<===> math/slash_as_division/warning
+DEPRECATION WARNING: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.
+Recommendation: math.div(2px, abs(1.5)) or calc(2px / abs(1.5))
+More info and automated migrator: https://sass-lang.com/d/slash-div
+  ,
+2 |   a: 2px / abs(1.5);
+  |      ^^^^^^^^^^^^^^
+  '
+    input.scss 2:6  root stylesheet
+
+<===>
+================================================================================
+<===> error/incompatible_units/input.scss
+a {b: abs(17px + 4deg)}
+
+<===> error/incompatible_units/error
+Error: 17px and 4deg are incompatible.
+  ,
+1 | a {b: abs(17px + 4deg)}
+  |           ^^^^^^^^^^^
+  '
+  input.scss 1:11  root stylesheet
```

Comment:
As above, this is just testing the simplification logic which is already covered by the `calc()` tests.